pyproject.toml

```diff
@@ -16,6 +16,7 @@ classifiers = [
 requires-python = ">=3.10"
 dependencies = [
   "typer>=0.12,<0.13",
+  "pyarrow>=12.0.0",
```

Comment:
how large is pyarrow?

Reply:
Hmmm, quite big, about 100MB. I guess it brings some arrow binaries with it. Should i better make an extra (parquet) for it?